docs/content/en/docs/topics/accessing-services/stargate/_index.md

```diff
@@ -6,39 +6,141 @@ description: |
   Accessing the K8ssandra Stargate interfaces.
 ---
 
-[Stargate](https://stargate.io/) is an open-source framework providing common
+[Stargate](https://stargate.io/) is an open source data gateway providing common
 API interfaces for backend databases. With K8ssandra, Stargate may be deployed
-in front of the Apache Cassandra cluster providing CQL, REST, and GraphQL API
-endpoints. These endpoints may be scaled horizontally independently of data
-layer scaling needs. This guide covers accessing the various API endpoints
+in front of the Apache Cassandra cluster providing CQL, REST, GraphQL, and document based API
+endpoints. These endpoints may be scaled independently of the data
```

Comment:
The double adverb was awkward, but I think "horizontally" was contributing useful value to the sentence -- we're talking about increasing the number of Stargate endpoints, not increasing the resources allocated to each endpoint. I don't have a good suggestion offhand for a rephrase, but let's spend a minute and see if we can work that tidbit back in here.

Reply:
How about `Stargate itself may be scaled horizontally within the cluster as needed.  This scaling is done independently of the data layer scaling.`?  With a link to the section for scaling the data layer?

Reply:
Updated this in the last commit to bring back the note about the independent scaling.